src/components/SignUpPage/DoB.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {createRoot} from 'react-dom'
+import "react-datepicker/dist/react-datepicker.css";
+import DatePicker from "react-datepicker";
+import Form from "react-bootstrap/Form";
+import {withFormik} from "formik";
+
+const today = new Date();
+
+export const DoB = props => {
+  const {
+    values,
+    touched,
+    errors,
+    handleSubmit,
+    setFieldValue,
+    setFieldTouched
+  } = props;
+
+  return (
+    <div>
+      <Form onSubmit={handleSubmit}>
+        <Form.Group controlId="DateOfBirth">
+          <Form.Label>Date of Birth</Form.Label>
+
+          <DatePicker
+            selected={values && values.date ? values.date : null}
+            onChange={(e) => {
+              setFieldValue('date', e);
```

Comment:
When clicking on any date inside date picker I see the following error. Could you please fix?
```
DoB.js:29 Uncaught TypeError: setFieldValue is not a function
    at onChange (DoB.js:29:1)
    at r.setSelected (react-datepicker.min.js:1:1)
```